addons/mail/views/mail_activity_views.xml

```diff
@@ -200,6 +235,10 @@
         <field name="priority">32</field>
         <field name="arch" type="xml">
             <form string="Log an Activity" create="false" delete="false">
+                <header>
+                    <button string="Mark as Done" class="btn-primary"
+                            name="action_done_other_activities" type="object"/>
```

Comment:
Maybe (even if verbose, otherwise I thought it was used to mark other activities as done)
```suggestion
                            name="action_done_redirect_to_other" type="object"/>
```